files/lang/ru.po

```diff
@@ -1173,7 +1173,7 @@ msgstr "Божественный Доспех"
 
 #, fuzzy (18 char max)
```

Comment:
Could you remove the `#, fuzzy` comments after these lines, please?  Even with the `(18 char max)`, I'm afraid the following line wouldn't get included.  Please also check for any nearby `#, fuzzy` comments, too.